.github/CODEOWNERS

```diff
@@ -12,6 +12,7 @@ shell.nix             @status-im/devops
 
 # Custom ownership for specific folders
 # Feel free to add yourself for any new packages you implement.
+/_docs/policies       @status-im/status-go-guild @iurimatias @alaibe @shivekkhurana @ilmotta
```

Comment:
Even if we have "Require review from code owners" enabled (which we don't), it will still only require 1 person from this list to approve. Yet it will request review from everybody from the list.

So it's not about _requirement_, but about _notifications_.

Reply:
This raises an interesting question, instead of version controlling the specific team leads what about creating a status team lead team eg `@status-im/status-leads`. Then we don't need to submit a PR to change the people that get pinged.

Reply:
@Samyoul I've thought about that when proposing to add this.
Thing is that `@status-im/status-leads` would also include web, keycard and network leads, which we doesn't need. So we'd need `@status-desktop-leads` and `@status-mobile-leads`, which would each contain just 2 people and is sounds like overcomplicating instead of making it simpler 😄